transport-udt/src/main/java/io/netty/channel/udt/nio/NioUdtByteConnectorChannel.java

```diff
@@ -186,4 +190,33 @@ public InetSocketAddress localAddress() {
     public InetSocketAddress remoteAddress() {
         return (InetSocketAddress) super.remoteAddress();
     }
+
+    private static void privilegedBind(final SocketChannelUDT socketChannel, final SocketAddress localAddress)
```

Comment:
As I mentioned in my comment below, I believe bind will have to stay here until com.barchart.udt.barchart-udt-bundle is released built on Java 7.